drivers/video/Kconfig.ov2640

```diff
@@ -0,0 +1,10 @@
+# OV2640
+
+# Copyright (c) 2021 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
The drivers/video/ov2640_regs.h file  has a different license (MIT), could you clarify this?

Reply:
I've done some changes to use Apache 2.0 license.